openapi/v2023.8.2/paths/serviceReadOrganisation.yaml

```diff
@@ -0,0 +1,19 @@
+deprecated: false
+description: Read organization
+operationId: readOrganisation
```

Comment:
`operationId` should be the same as the file name. Auto code generators will use this as the function name.